IdentitySdkCore/IdentitySdkCore/Classes/ReachFiveApi.swift

```diff
@@ -367,4 +367,18 @@ public class ReachFiveApi {
                 .validate(contentType: ["application/json"])
                 .responseJson(type: AuthenticationToken.self, decoder: self.decoder)
         }
+    
+    public func getWebAuthnRegistrations(authorization: String) -> Future<[DeviceCredential], ReachFiveError> {
```

Comment:
The caller has to build a `String` to pass in the `Authorization` header, I'd prefer this method to take the `AuthToken` directly and maybe signal an error if the token type is not right (we should have better typing but this will have to wait), especially since the method is `public`.